src/api/app/controllers/webui/cloud/azure/configurations_controller.rb

```diff
@@ -6,9 +6,7 @@ class ConfigurationsController < WebuiController
         before_action :set_azure_configuration
 
         # GET /cloud/azure/configuration
-        def show
-          switch_to_webui2
-        end
+        def show; end
```

Comment:
An empty action isn't needed because of [Rails' default](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-by-default-convention-over-configuration-in-action)

There are a few cases of this in the PR. I didn't comment on every single case to avoid spamming the same comment over and over.

Reply:
@dmarcoux, I removed some of them. The others, rubocop complains, because they are called in a `before_action` and it's need to be explicitly defined.

Reply:
What was the RuboCop offense? I'm curious.

Reply:
https://rails.rubystyle.guide/#lexically-scoped-action-filter